generated_codegen.md

```diff
@@ -504,43 +633,52 @@ Uses of `ExecutableWriterHolder` and `ExecutableWriterHolderNoLog` under `src\co
   50:             ExecutableWriterHolder<void> slotWriterHolder((void*)slot, sizeof(PCODE*));
   51              *(PCODE *)slotWriterHolder.GetRW() = entryPoint - ((PCODE)slot + sizeof(PCODE));
   52              // fall through
-
 ```
 
-`src\coreclr\vm\precode.cpp`:
+## `src\coreclr\vm\precode.cpp`:
+
+`Precode::Allocate` - Only for `PRECODE_THISPTR_RETBUF`.
```

Comment:
It seems to be used for more - call counting stub, func ptr stubs, fixup precode, stub precode, ndirect import precode